### ISSUE TITLE
Create a dark theme for the editor (not the chart)

### ISSUE THREAD

**Issue by ArexJambusarwala** [vega/editor#295]
We can provide an option to choose from a light/dark them for the editor.

**Comment by domoritz** [comment]
While this might be nice, it's not a priority.

**Comment by nik72619c** [comment]
Shall I work on this one ?

**Comment by algomaster99** [comment]
@nik72619c this is a very tough issue to start with. In fact, it can be one full project in itself. Anyway, if you have designs for it, you may share it here. 🙂

**Comment by nik72619c** [comment]
Sure,I'll givenit a try

**Comment by domoritz** [comment]
Before we start on a dark theme, we should refactor the CSS. I think one good way to do that would be to use for example tachyons and a limited set of colors.

**Comment by domoritz** [comment]
Closing as it's not the highest priority item and adds more overhead for maintenance.
